apps/chat/src/components/Files/SelectFolderModal.tsx

```diff
@@ -117,10 +119,11 @@ export const SelectFolderModal = ({
             loadingFolderIds: loadingFolderIds,
             featureType: FeatureType.File,
           }}
-          handleToggleFolder={handleToggleFolder}
+          handleFolderSelect={handleFolderSelect}
           isAllEntitiesOpened={isAllFilesOpened}
           selectedFolderId={selectedFolderId}
           rootFolderName="All files"
+          rootFolderId={getRootId()}
```

Comment:
is `SelectFolderModal` used only for `files`?

Reply:
Yes

Reply:
Yes